themes/10up-block-theme/includes/core.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Core setup, site hooks and filters.
+ *
+ * @package TenupBlockTheme
+ */
+
+namespace TenupBlockTheme\Core;
+
+use function TenupBlockTheme\Utility\get_asset_info;
+
+/**
+ * Set up theme defaults and register supported WordPress features.
+ *
+ * @return void
+ */
+function setup() {
+	$n = function ( $function_name ) {
+		return __NAMESPACE__ . "\\$function_name";
+	};
```

Comment:
**thought (non-blocking):** It seems like this is unused now (🤞) Let's remove it anyway and use the full namespace as we are below in all-the-places

Reply:
Yeah this should be removed. I used the full namespace so we get better IDE integration (Props @tobeycodes )

Reply:
Thank you! <3 IDE integration